docs/pages/contributing/documentation/best-practices.mdx

```diff
@@ -0,0 +1,86 @@
+
+Whenever possible, choose words that have one clear meaning. When in doubt, choose the 
+simple word or phrase over a more formal or complex one.
+Avoid flowery language and unnecessary words whenever possible.
+For example:
+
+| Choose this | Instead of this |
+| --- | --- |
+| use | utilize, make use of |
+| remove | extract, take away, eliminate |
+| tell | inform, let know |
+| to | in order to, as a means to |
+| also | in addition |
+| connect | establish connectivity |
+
+Avoid adjectives and adverbs—words that describe how, when, or where. They're tempting, but 
+rarely add much value. Unless they're important to the meaning of a statement, leave them out.
+
+### Be consistent
+
+Use one term consistently to represent one concept. 
+For example, if you use tenant, cluster, and proxy interchangeably or ambiguously, you'll 
+leave the reader confused and uncertain. 
+If terminology changes, be prepared to root out old terminology.
+
+If you use words that can be both nouns and verbs—for example, words like file, post, mark, 
+screen, record, and report—use sentence structure and context to eliminate ambiguity.
+
+### Avoid dangling participles
+
+Participles are modifiers so they must have a noun to modify. A *dangling participle* is 
+a participle that doesn't have a noun to modify. If you misplace or leave out the word 
+being modified, you'll end up with a sentence that is difficult to understand, illogical, 
+or ambiguous (though potentially amusing). Here are a few examples of sentences with 
+dangling participles:
+
+- Looking around the yard, dandelions sprouted in every corner.
+- Walking through the kitchen, the smoke alarm was going off.
+- Driving like a maniac, the deer was hit and killed.
+
+You can correct these sentences by bringing the participle phrase closer to the subject 
+the phrase is intended to modify or changing the word order of the sentence to clarify 
+who is doing what. You can also fix these types of problems by changing the tense or 
+using the active voice. For example:
+
+- Looking around the yard, I saw dandelions had sprouted in every corner.
+- As I was walking through the kitchen, the smoke alarm was going off.
+- Driving like a maniac, he hit and killed the deer.
+
+### Dangling prepositions
+
+In modern English, it's perfectly acceptable to end a sentence with a preposition. 
+Don't twist a sentence into knots just to avoid using a preposition at the end.
+
+- This is something you might be interested in.
+- This is an example you should pay attention to.
+
+### Cross-reference formats
+
+Most cross-references should include information that clarifies what the reader 
+can expect to be found in the referenced topic.
+For cross-references to topics in the Teleport documentation, use the following 
+format:
+
+- For more information about task or concept, see topic-title-link.
+
+For cross-references in a glossary entry to other glossary entries, use the following 
```

Comment:
What counts as a glossary entry in the Teleport docs? Does this only apply to the Core Concepts page?